### PR TITLE
Labels

### PR DESCRIPTION
**Pull request by tchorzewski1991**
Labels controller and views:

Index:
label-index

Show:
label-show

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -98,11 +106,25 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "slug"
+    t.integer  "records_count"
   end
 
   add_index "labels", ["name"], name: "index_labels_on_name", unique: true, using: :btree
   add_index "labels", ["slug"], name: "index_labels_on_slug", unique: true, using: :btree
 
+  create_table "mugshots", force: true do |t|
```
Comment (holden) [inline db/schema.rb]: Errr... where did this thing come from.  I think @smadeja nuked it awhile back?

File: db/schema.rb [hunk 2]
```diff
@@ -83,6 +83,14 @@
     t.datetime "updated_at"
   end
 
+  create_table "bubbles", force: true do |t|
```
Comment (holden) [inline db/schema.rb]: Ditto here.  Pretty sure these tables should both be dead.

File: db/migrate/20160919075126_add_records_count_to_labels.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddRecordsCountToLabels < ActiveRecord::Migration
+  def change
+    add_column :labels, :records_count, :integer
```
Comment (holden) [inline db/migrate/20160919075126_add_records_count_to_labels.rb]: I'm not sure it makes a lot of sense to add a counter to the labels model.  In general I don't want to change the models unless you can make a compelling case.

File: db/schema.rb (hunk 2 above)
Comment (holden) [inline db/schema.rb]: @smadeja It's a little bit worrying, how did these come back?  (assuming your nukes were fully engaged)

File: db/schema.rb (hunk 1 above)
Comment (tchorzewski1991) [inline db/schema.rb]: I have no idea where it comes from :/ Sorry, didn't see that earlier ...

File: db/migrate/20160919075126_add_records_count_to_labels.rb (hunk 3 above)
Comment (tchorzewski1991) [inline db/migrate/20160919075126_add_records_count_to_labels.rb]: ok, I will remove it. Just want to make it as optimalized as possible.

**Comment by holden** [comment]
Now that you've removed it i'm getting.  `undefined method `records_count' for #<Label:0x007f8c00b9e428>`

**Pull request merged**